bootstrap/postgres/configs.go

```diff
@@ -248,13 +267,26 @@ func (cr configRepository) RetrieveUnknown(offset, limit uint64) []bootstrap.Con
 		c := bootstrap.Config{}
 		if err = rows.Scan(&c.ExternalID, &c.ExternalKey); err != nil {
 			cr.log.Error(fmt.Sprintf("Failed to read retrieved config due to %s", err))
```

Comment:
Failed to retrieved config due to

Reply:
@manuio I've explained why I prefer to keep this message here https://github.com/mainflux/mainflux/pull/575#discussion_r256784476. If you insist to change, no problem, it's a simple update.